src/components/edit/Interactions.jsx

```diff
@@ -75,12 +115,12 @@ export default class EditInteractions extends Component {
     hiQuery.agentId = this.state.hiAgent.id;
     hiQuery.oakId = this.state.hiOak.id;
     getInteractionsByOakAndAgent(hiQuery)
-      .then(interaction => this.setState({ hi: interaction }));
+      .then(interaction => this.setState({ hi: interaction, hiSymptoms: interaction.hiSymptoms }));
   }
 
   render() {
```

Comment:
This file is getting quite large, we may want to pull the render out into a stateless functional component and make this a container instead.